### DISCUSSION TITLE
微调后模型如何使用？

### DISCUSSION THREAD

**Discussion by KevinFanng** [zai-org/ChatGLM3#830] [Q&A] [answered]
官方提供的“在本仓库的其他 demo 或者外部仓库使用微调后的模型”
![在本仓库的其他 demo 或者外部仓库使用微调后的模型](https://github.com/THUDM/ChatGLM3/assets/85269058/02044082-9d6a-4651-8f92-a3d934a256e7)
那部署的时候，是不是需要将ChatGLM3/finetune_demo/output/checkpoint-3000这个目录及"base_model_name_or_path": "/mnt/workspace/chatglm3-6b/"这两个目录都拷贝走，就可以了？
![微调模型](https://github.com/THUDM/ChatGLM3/assets/85269058/af90eb83-8da3-4560-aac3-68f151927547)
![基座模型](https://github.com/THUDM/ChatGLM3/assets/85269058/786a1b5d-160b-45ed-8e78-7244aa3baced)

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
只要是支持peft的都是一样的，会自动的挂载
